tasks/build/release.cr

```diff
@@ -6,20 +6,19 @@ class Build::Release < LuckyCli::Task
 
   summary "Compile app for production"
 
-  def initialize(@io : IO = STDOUT, @error_io : IO = STDERR)
-  end
+  property error_io : IO = STDERR
 
   def call
-    command = "crystal build --release src/start_server.cr"
+    command = "crystal build --release src/start_server.cr -o bin/start_server"
 
     log "Building binary with '#{command}"
```

Comment:
Why does this print out, but the one below doesn't?

<img width="1164" alt="Screen Shot 2021-04-02 at 2 46 41 PM" src="https://user-images.githubusercontent.com/2391/113456511-41b12900-93c2-11eb-9d25-5bd833111a90.png">

Reply:
Maybe try adding an else and check if the process isn't completing successfully?

Reply:
ah. Good call. It is failing.. 🤔  but I guess why.

Reply:
oh, but that still doesn't explain why `IO::Memory` is printing to STDOUT 🤨

Reply:
Ohh, it's actually the second spec that is printing to stdout, it doesn't have it's output changed, only error_io

Reply:
ooohhhh 😂  yup. That fixes it. Thanks!